### ISSUE TITLE
Streamlink 1.2.0

### ISSUE THREAD

**Issue by bastimeyer** [streamlink/streamlink#2583]
I think we should publish a new release. There are a lot of changes and crucial plugin fixes since the last one in April.

Any open and unresolved blockers? Anything that should be merged first? Does anybody want to write a short changelog? Optional though, as we include the git shortlog anyway.

**Comment by nsmith1024** [comment]
Has this issue been fixed

**Comment by bastimeyer** [comment]
This is unrelated off-topic spam. Everybody subscribed to the issue tracker will see your other posts. This thread is about the next release. If you're having a question regarding unresolved issues/bugs or feature requests, post a comment in the according thread.

**Comment by nsmith1024** [comment]
Oh sorry i wasnt trying to SPAM, where do i ask this question? I posted a question in the issues and it seems to have disappeared, so where should i post my question? thanks!

**Comment by bastimeyer** [comment]
> Where do users post questions about using streamlink?

You can open a new issue on the issue tracker here or ask in the Gitter/IRC channel. But you already did that and I answered it and closed the thread, as it's out of scope of Streamlink.

> I posted a question and it was removed

You posted "I was thinking of using streamlink for a twitch project, but now read that v3 will stop. Does streamlink have v5 capability already? anybody knows?" in #2571 and deleted it yourself. I can see that in the project's audit log. Btw, this question was already answered by me in this thread. v3 is irrelevant and if you're having questions about the Twitch API, then you should read the developer documentation on Twitch.

**Comment by nsmith1024** [comment]
Oh thanks for answering it, i thought the issue i created was deleted because i didnt see it in the issues list anymore. Found out that when an issue is closed its removed from the issues, so then i didnt see your answer.
